release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -479,6 +479,18 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+* Running the `operator-sdk new` command without the `--helm-chart` flag builds
+and runs a Helm-based Operator that uses the default boilerplate Nginx chart.
+While this example chart works correctly on upstream Kubernetes, it fails to
+deploy successfully on {product-title}. To work around this issue, use the
+`--helm-chart` flag to provide a Helm chart that deploys successfully on
+{product-title}.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
+
+[id="ocp-4-6-asynchronous-errata-updates"]
+== Asynchronous errata updates
```

Comment:
@codyhoag @jeana-redhat FYI adding this here since it seemed to be missing.

Reply:
The usage for the Helm-based Operator is ` ./helm-operator run -- --helm-chart`

Reply:
@jianlinliu Checked with jlandford that the workaround should be referring to using the `--helm-chart` flag with the `operator-sdk new` and `operator-sdk create api` commands.